build.sbt

```diff
@@ -68,24 +64,9 @@ val scalaUriSettings = Seq(
       case _ =>
         "org.parboiled" %%% "parboiled" % "2.2.0"
     },
-    "com.chuusai"   %%% "shapeless"  % "2.3.3",
-    "org.typelevel" %%% "simulacrum" % "1.0.0" % Provided,
-    "org.typelevel" %%% "cats-core"  % "2.1.1"
-  ),
-  libraryDependencies ++= paradisePlugin.value,
-  pomPostProcess := { node =>
```

Comment:
I'm going to bring back a similar `pomPostProcess` after merging. Without this we get the dependency below in our pom which isn't needed at runtime. I've tested removing this dep from the pom with JVM and Scala.js version and it seems fine. Can also see that cats are [doing the same thing](https://github.com/typelevel/cats/commit/6f27f07eb44dd09a28767e35e3fa2a2f54085153)

```xml
        <dependency>
            <groupId>org.typelevel</groupId>
            <artifactId>simulacrum-scalafix-annotations_sjs1_2.13</artifactId>
            <version>0.1.0</version>
        </dependency>
```